packages/atoms/src/components/kiwi-checkbox-dropdown-menu/kiwi-checkbox-dropdown-menu.stories.js

```diff
@@ -29,3 +29,31 @@ basic.args = {
   buttonText: 'Click me',
   options: ['Option 1', 'Option 2', 'Option 3'],
 };
+
+export const rightScreenEdge = (args) => {
+  const { buttonText, options, menuEntryWidth } = args;
+
+  return html`<div class="panel panel-default m-1">
+    <div class="panel-body" style="text-align:right">
```

Comment:
```suggestion
    <div class="panel-body pull-right">
```
Ich hoffe das funktioniert so, ansonsten musst du die Klasse `pull-right` mal bei einem Kindelement setzen.